s3-eventbridge-wildcard-sam/example-pattern.json

```diff
@@ -0,0 +1,61 @@
+{
+  "title": "Wildcard pattern match in Amazon EventBridge rule",
+  "description": "The SAM template deploys an Amazon S3 bucket, an Amazon EventBridge rule with wildcard pattern match and an Amazon EventBridge target as Amazon SQS Queue.", 
+  "language": "YAML",
+  "level": "200",
+  "framework": "SAM",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "This template creates an S3 bucket that publishes events to Amazon EventBridge, an EventBridge rule with wildcard filter to select only the desired events from dynamic subfolders in Amazon S3 bucket and also an Amazon SQS Queue as the target.",
+      "Once the stack is deployed, we will use upload-to-s3.sh to create dynamic subfolders based on current date and timestamp within the S3 bucket and upload new files. ",
+      "This will trigger multiple events, however, only the events that will match the wildcard pattern as per the EventBridge rule, will be selected and a message will be sent to the SQS Queue.",
+      "We will inspect the messages from the SQS Queue to validate the same."
+    ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/s3-eventbridge-wildcard-sam",
+      "templateURL": "serverless-patterns/s3-eventbridge-wildcard-sam",
+      "projectFolder": "s3-eventbridge-wildcard-sam",
+      "templateFile": "template.yaml"
+    }
+  },
+  "resources": {
+    "bullets": [
```

Comment:
Please include a link to S3 event notifications

Reply:
Added.